help/de/Help_para-assist-buttons.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ich frage mich, ob es Sinn macht, das noch was zu erweitern zu <q>Grundlegendes zur Benutzeroberfläche</q>:
Der Aktionsbereich enthält Schaltflächen eines Dalogs oder Assistenten Er befindet sich

- normalerweise am unteren Rand
- unter Ubuntu's Unity in der Titrelleiste
- unter MacOS ?
- 
Normalerweise sind die folgenden Schaltflächen vorhanden
<glosslist…>

- immer
Hilfe
Abbrechen, …
- beim normalen Dialog …
- in Assistenten …

Aber das können wir bei Bedarf noch anpassen. Mit geschickter Begriffs-Hierarchie sollten da auch Referenzierungen möglich sein

Reply:
> * normalerweise am unteren Rand
> * unter Ubuntu's Unity in der Titrelleiste
> * unter MacOS ?

Das möchte ich nicht so gerne aufdröseln, dann rennen wir hinter jedem neuen Windowmanager her. Außerdem findet der Anwender die Buttons schon selber im Dialogfenster.

Reply:
Darum ging es jetzt garnicht. `normalerweise am unteren Rand, aber einige Fenstermanager handhaben das anders.` reicht. Eine Abgrenzung von und ein Verweis in den entspechenden Bereich der Gnome-Doku überlasse ich dir..

Reply:
jetzt verstehe ich wo du hin willst!
Einmal alle mögliche Schaltflächen aufzählen und beschreiben und dann bei Beschreibung der Assistenten pauschal auf diese Erläuterungen verweisen.

Reply:
Genau!